src/services/auth/index.tsx

```diff
@@ -119,6 +126,10 @@ export const liveValidateOTP = async (
   endpoint: string
 ): Promise<SessionCredentials> => {
   const payload = { code, otp, phone: mobileNumber };
+  Sentry.addBreadcrumb({
+    category: "otp validate",
```

Comment:
hmm, we currently use camelCase when writing the category haha. what if we change the category name to `validateOtp` instead haha

Reply:
or `otpValidate` haha